### ISSUE TITLE
Many warnings appeared while generating report!

### ISSUE THREAD

**Issue by mingjun1120** [fbdesignpro/sweetviz#112]
![image](https://user-images.githubusercontent.com/54136320/157420833-d46a6ef0-9184-4573-8301-c9e704e687aa.png)

**Comment by vincenzomanzoni** [comment]
You can disable it with:

```
import warnings
warnings.filterwarnings("ignore", category=FutureWarning)
```

Sweetviz team: do you have in the roadmap to fix this warning in the codebase?

**Comment by fbdesignpro** [comment]
@mingjun1120 , @vincenzomanzoni thank you for the report, I will look into it ASAP!

**Comment by fbdesignpro** [comment]
Apologies for the delay, warnings should be fixed in the latest version 2.1.4!
